pg2kinesis/__main__.py

```diff
@@ -68,6 +72,7 @@ def __init__(self, formatter, writer):
         self.msg_window_size = 0
         self.msg_window_count = 0
         self.cur_window = 0
+        self.last_flush = datetime.now()
```

Comment:
How is `self.last_flush` being used in the code?

Reply:
Was used to force a flush every 30 seconds but as that is not our approach now I have removed it. 
Good catch. :)